javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -8,8 +8,8 @@ const BasicCourseTable = ( {classes} ) => {
   const rowStyle = (row, _rowIndex) => {
     return  (row.section % 100 === 0)? {backgroundColor: '#CEDEFA'}: {backgroundColor: '#EDF3FE'};
   }
-  const dataAlignment = (_cell, row) => {
-    const alignmnet = (row.section % 100 === 0)? 'left': 'right';
+  const dataAlignment = () => {
+    const alignmnet = 'left';
     return alignmnet
```

Comment:
This isn't an issue that was created by your team, but can we fix the spelling of the variable name "alignment" here? 

Also, if this is always going to return "left", is there a point to keeping this `dataAlignment` function at all? We can just hard-code the value "left" instead of calling `dataAlignment` a bunch of times everywhere. In fact, since I think the table is set by default to align left, we might not even need the `align` field anymore.

Reply:
Agreed with Andrew.. it's a small fix, so please go ahead and make it.